src/libConsensus/ConsensusCommon.h

```diff
@@ -80,7 +72,7 @@ class ConsensusCommon
     State m_state;
 
     /// The minimum fraction of peers necessary to achieve consensus.
-    const double TOLERANCE_FRACTION;
+    static constexpr double TOLERANCE_FRACTION = (double) 0.667;
```

Comment:
0.667 is already double literal, probably we can drop the conversion. See http://en.cppreference.com/w/cpp/language/floating_literal

Reply:
Agree. Pushed update.